plugin/sdk_v2/shared/core/bus_context.py

```
@@ -0,0 +1,746 @@
+            if isinstance(item, Mapping)
+        ]
+        return SdkBusList(
+            items,
+            namespace=self._namespace,
+            record_factory=self._record_factory,
+            host_ctx=self._host_ctx,
+            raw_list=value,
+        )
+
+    def get(self, *, bucket_id: str, limit: int = 20, timeout: float = 5.0) -> object:
+        method = getattr(self._raw_namespace, "get")
```

Ruff (0.15.5)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)